churaverse-plugins-client/src/synchroBreakPlugin/domain/nyokkiLogTextCreator.ts

```diff
@@ -0,0 +1,91 @@
+import { Store, IMainScene } from 'churaverse-engine-client'
+import { INyokkiLogTextCreator } from '../interface/INyokkiLogTextCreator'
+
+const successLog = [
+  'はnyokkiに成功しました',
+  '素晴らしい!!、nyokki成功',
+  'nyokki成功。ナイス!!',
+  'nyokkiミッション成功',
+  'は見事にnyokkiを達成しました',
+  'nyokki達成お見事',
+  '最高のnyokki成功',
+  '完璧なnyokki',
+  'nyokki成功',
+  'nyokki成功、賞金ゲット!!',
+]
```

Comment:
完全な定数なので、中身を書き換えられないようにas constをつけるといいかなと思います
```suggestion
] as const
```